checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
+            "text": "Avoid wildcards as sources for DNATS",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "c44c6f0e-1642-4a61-a17b-0922f835c93a",
+            "id": "D07.13",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/tutorial-firewall-dnat"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Prevent SNAT Port exhaustion",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "7371dc21-251a-47a3-af14-6e01b9da4757",
+            "id": "D07.14",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/nat-gateway/tutorial-hub-spoke-nat-firewall"
+        },
+         {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable TLS Inspection",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "346840b8-1064-496e-8396-4b1340172d52",
+            "id": "D07.15",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#tls-inspection"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Consider Web Categories to allow or deny outbound access in bulk.",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "39990a13-915c-45f9-a2d3-562d7d6c4b7c",
+            "id": "D07.16",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#web-categories"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use Azure Firewall in parallel/sequence with Azure App Gateway",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "6eff7e6c-6c4a-43d7-be3f-6641c2cb3d4a",
+            "id": "D07.17",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/firewall-application-gateway#firewall-and-application-gateway-in-parallel"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable Azure Firewall DNS proxy configuration ",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "94f3eede-9aa3-4088-92a3-bb9a56509fad",
+            "id": "D07.18",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/dns-details"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
```

Comment:
This isn't a firewall requirement, so it might belong in a different section.  The ALZ guidance is to have a policy denying public IPs - we should check if that policy is covered here, and address there.